<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YESNOMessageBox</class>
 <widget class="QDialog" name="YESNOMessageBox">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>328</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Phils-Hub - YES/NO Message Box</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip {
    background-color: #2e2e2e;
    color: #ffffff;
    border: 1px solid #5b5e60;
    padding: 4px;
    border-radius: 4px;
    opacity: 200;
    font-size: 12px;
}

QLabel {
    color: #ffffff;
}

QFrame {
    background-color: #2e2e2e;
}

QPushButton {
    background-color: #3c3f41;
    color: #ffffff;
    border: 1px solid #5b5e60;
    border-radius: 4px;
	padding: 0px 0px;
}

QPushButton:hover {
    border-color: #4a90e2;
}

QPushButton:pressed {
    background-color: #3d78b2;
    border-color: #2a5f92;
}

/*
red: ff050d
green: 0dce1d
*/</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border-top-left-radius: 9px;
	border-top-right-radius: 9px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources.qrc">:/ico/Icons/ico/phils-hub.ico</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="msg_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>YES/NO Message Box...</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">QFrame {
	background: transparent;
	border-bottom-left-radius: 9px;
	border-bottom-right-radius: 9px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="yes_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-bottom-right-radius: 0px;
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
}</string>
        </property>
        <property name="text">
         <string>YES</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="no_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-bottom-left-radius: 0px;
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
}</string>
        </property>
        <property name="text">
         <string>NO</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
